data/data_test.go

```diff
@@ -120,7 +114,7 @@ func (s *DiskvSpoolerTestSuite) TestSpoolData(t *C) {
 
 	// Doesn't matter what data we spool; just send some bytes...
 	now := time.Now()
-	logEntry := &proto.LogEntry{
+	logEntry := proto.LogEntry{
```

Comment:
Turns out there was a change to pass everywhere `proto.LogEntry` as value instead of passing it by reference.